### ISSUE TITLE
Script to generate filelist.cmake does not work on macos

### ISSUE THREAD

**Issue by MathiasPaulin** [STORM-IRIT/Radium-Engine#989]
On macOS Big Sur Version 11.6.8, using apple provided bash version
```
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin20)
Copyright (C) 2007 Free Software Foundation, Inc.
```
and with cmake-format version 0.6.13,

the script generateFilelistForModule.sh produces wrong output.

Script log : 
```
./generateFilelistForModule.sh Gui      
./generateFilelistForModule.sh: line 9: ${BASE,,}: bad substitution
generate [Gui] filelist
-- input files from         [../src/Gui]
-- output vars prefix       []
-- output file is           [../src/Gui/filelist.cmake]
```

Generated file (snippet)
```
# ----------------------------------------------------
# This file can be generated from a script:
# To do so, run "./generateFilelistForModule.sh Gui"
# from ./scripts directory
# ----------------------------------------------------

set(_sources
    /AboutDialog/AboutDialog.cpp
    /BaseApplication.cpp
    /ParameterSetEditor/MaterialParameterEditor.cpp
    /ParameterSetEditor/ParameterSetEditor.cpp
...
``` 

Expected result (same snippet)
```
# ----------------------------------------------------
# This file can be generated from a script:
# To do so, run "./generateFilelistForModule.sh Gui"
# from ./scripts directory
# ----------------------------------------------------

set(gui_sources
    AboutDialog/AboutDialog.cpp
    BaseApplication.cpp
    ParameterSetEditor/MaterialParameterEditor.cpp
    ParameterSetEditor/ParameterSetEditor.cpp
...
```

**Comment by dlyr** [comment]
`${BASE,,}: bad substitution` seems related to bash, 
you could run 
`$ bash ./generateFilelistForModule.sh Gui`
I open a PR to fix the prefix / (find seems to have different behaviors on macos)

**Comment by MathiasPaulin** [comment]
Closed by #990
